understanding/identify-input-purpose.html

```diff
@@ -58,198 +56,137 @@
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.3.5:</span>Identify Input Purpose (Level AA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.3.5:</span>入力目的の特定 (レベル AA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>The purpose of each input field collecting information about the user can be <a href="#dfn-programmatically-determined">programmatically determined</a> when:
+                  <p>利用者の情報を集める入力フィールドのそれぞれの目的は、次の場合に<a href="#dfn-programmatically-determined">プログラムによる解釈</a>が可能である:
                   </p>
                   <ul>
                      
-                     <li>The input field serves a purpose identified in the <a href="#input-purposes">Input Purposes for user interface components section</a>; and
+                     <li>入力フィールドが、<a href="https://waic.jp/translations/WCAG22/#input-purposes">ユーザインタフェース コンポーネントの入力目的の節</a>で示される目的を提供している、かつ
                      </li>
                      
-                     <li>The content is implemented using technologies with support for identifying the expected
-                        meaning for form input data.
+                     <li>フォーム入力データとして想定される意味の特定をサポートする技術を用いて、コンテンツが実装されている。
                      </li>
                      
                   </ul>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>It is easier to fill out forms.</dd>
+                  <dt>目標</dt>
+                  <dd>フォームへの記入を、より簡単にできるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Use code to indicate the purpose of common inputs, where technology allows.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>技術が許す限り、コードを用いて共通の入力の目的を示す。</dd>
```

Comment:
「許す限り」というのは違和感があります。この where は if のように訳してしまって良いのではないでしょうか。

```suggestion
                  <dd>技術が許すならば、コードを用いて共通の入力の目的を示す。</dd>
```